src/main/java/org/radarcns/webapp/filter/CorsFilter.java

```diff
@@ -0,0 +1,24 @@
+package org.radarcns.webapp.filter;
+
+import javax.annotation.Priority;
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerResponseContext;
+import javax.ws.rs.container.ContainerResponseFilter;
+import javax.ws.rs.core.MultivaluedMap;
+import javax.ws.rs.ext.Provider;
+
+/** Standard CORS filter, applied to all responses. */
+@Provider
+@Priority(1000)
+public class CorsFilter implements ContainerResponseFilter {
```

Comment:
Doesn't this has to be registered to the context as well? Or is it registered by default?

Reply:
Filters with an `@Provider` tag, in a directory that is scanned by `RadarApplication`, are automatically added.